Makefile.cache

```diff
@@ -0,0 +1,637 @@
+#######################################################################
+#
+#   Copyright (c) 2020 Broadcom, Inc.
+#   The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
+#
+#   Author: Kalimuthu Velappan
+#   Email: kalimuthu.velappan@broadcom.com
+#  
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#  
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#######################################################################
+#
+#
+#            DPKG caching framework
```

Comment:
Do we need to add the .dep files for the other platforms, such as innvium, mellanox, etc?

Reply:
Yes, we are working on this. We will address this in next phase.